_posts/2023-02-22-solidity-0.8.19-release-announcement.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.19 Release Announcement'
+date: '2023-02-22'
+author: Solidity Team
+category: Releases
+---
+
+Introducing the newest version of the Solidity Compiler!
+
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19). 
+This latest version includes a range of improvements and it also introduces support defining *operators* on [user defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs)!
+
+### Notable New Features
+These are the features we want to highlight in this release.
+
+#### Operators for user defined value types
+The ability to use operators is meant to bring the UDVTs closer to being as natural to use
+as the built-in types and promote the use of type-safe interfaces.
+
+TODO You can read more on @cameel's latest blogpost
+
+## Full Changelog
+### Language Features
+* TODO
+
+### Compiler Features
+* TODO
```

Comment:
```suggestion
* SMTChecker: New trusted mode that assumes that any compile-time available code is the actual used code, even in external calls. This can be used via the CLI option ``--model-checker-ext-calls trusted`` or the JSON field ``settings.modelChecker.extCalls: "trusted"``.
```